nautobot/extras/models/jobs.py

```diff
@@ -301,6 +291,20 @@ def job_task(self):
         except TypeError as err:  # keep 2.0-2.2.2 exception behavior
             raise NotRegistered from err
 
+    @property
+    def task_queues(self):
+        return self.job_queues.all().values_list("name", flat=True)
```

Comment:
Have we lost the ability to have a specified order to these queues for a given job? What's the default ordering, just by `name`?

Reply:
Just by `name` yeah, will document this if this is an acceptable loss of functionality